test/events/component-view.test.js

```diff
@@ -158,7 +159,7 @@ describe('component:view', () => {
 						getContextData: {},
 					};
 
-					componentView.init(opts);
+					componentView(opts);
```

Comment:
did this have to change because we have introduced a change in the API?

Reply:
I think it's the import? Could be `componentView.init` with:

```diff
-import componentView from '../../src/javascript/events/component-view';
+import * as componentView from '../../src/javascript/events/component-view';
```